### PR TITLE
Upgrade QA tooling and move to PHP72

### PR DESCRIPTION
**Pull request by mroest**
In preparation to the SF4 upgrade this is the first step. This
updates the QA tooling using the best/common practices from the other
Stepup projects.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,40 +13,66 @@
...
-    "twig/extensions": "^1.5",
-    "symfony/assetic-bundle": "~2.3",
-    "symfony/monolog-bundle": "^3.1.0",
-    "sensio/distribution-bundle": "^5.0",
-    "sensio/framework-extra-bundle": "~3.0",
-    "incenteev/composer-parameter-handler": "~2.0",
-    "nelmio/security-bundle": "~1.4",
-    "mopa/bootstrap-bundle": "^3.2",
-    "twbs/bootstrap": "^3.3.0",
+    "php": "~7.2",
```
Comment (MKodde) [inline composer.json:16]: :metal:

File: .travis.yml [hunk 2]
```diff
@@ -8,8 +8,7 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
+  - 7.2
```
Comment (MKodde) [inline .travis.yml:11]: Consider adding the current PHP version (7.4) to see if we start introducing incompatibilities with that version during development.

File: composer.json [hunk 3]
```diff
@@ -13,40 +13,66 @@
...
   },
   "scripts": {
+    "test": [
+      "@lint",
+      "@phpmd",
+      "@phpcs",
+      "@phpcpd",
+      "@phpunit",
+      "@behat",
+      "@security-tests"
```
Comment (MKodde) [inline composer.json:59]: While at it, why not add the docheader qa check? See stepup-tiqr for an implementation example.

File: .travis.yml [hunk 4]
```diff
@@ -21,18 +20,17 @@ cache:
...
 
 branches:
   only:
     - master
     - develop
-
-addons:
-  apt:
-    packages:
-      - ant
```
Comment (MKodde) [inline .travis.yml:38]: :heart:

**Review by MKodde — CHANGES_REQUESTED** [review]
Aside from the docheader question, looks good to go!

File: .travis.yml (hunk 2 above)
Comment (mroest) [reply .travis.yml:11]: Yes, but lets do that in a different PR.

File: composer.json (hunk 3 above)
Comment (mroest) [reply composer.json:59]: Sure, but lets do it in a different PR. I do not want to include too much unrelated stuff and only stick to the php7.2 upgrade

**Review by MKodde — APPROVED** [review]

**Pull request merged**